app/containers/wallet/WalletSummaryPage.js

```diff
@@ -76,15 +80,40 @@ export default class WalletSummaryPage extends Component<Props> {
       }
     }
 
+    const notification = this._getThisPageActiveNotification();
+
     return (
       <VerticalFlexContainer>
+
+        <NotificationMessage
+          icon={successIcon}
+          show={!!notification.id}
+        >
+          {notification.id && <FormattedHTMLMessage {...notification.message} />}
+        </NotificationMessage>
+
         <WalletSummary
           numberOfTransactions={totalAvailable}
           pendingAmount={unconfirmedAmount}
           isLoadingTransactions={recentTransactionsRequest.isExecutingFirstTime}
         />
+
         {walletTransactions}
+
       </VerticalFlexContainer>
     );
   }
+
+  _getThisPageActiveNotification = (): Notification | any => {
```

Comment:
why `any`?
Maybe use `?Notification`

Reply:
yeah, we can use `?Notification`
I was trying some other way, just forget to remove `any` 
will update it.